content/docs/user-guide/how-to/undo-dvc-add.md

```diff
@@ -0,0 +1,47 @@
+# Undo dvc add
+
+There are situations where you `dvc add` a data file by mistake and want DVC to
+stop tracking that file. Follow the steps listed here to undo `dvc add`.
+
+Lets first add a data file into our example <abbr>project</abbr>:
+
+```dvc
+$ dvc add data.csv
+$ tree
+.
+├── data.csv
+└── data.csv.dvc
+```
+
+As you can see `dvc add` creates a `.dvc` file to track the added data. Now
+let's reverse this action.
+
+In the first step, if you are using `symlink` or `hardlink` as
+[link](doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
+type for DVC <abbr>cache</abbr>, you will have to unprotect the tracked file
+(see `dvc unprotect`):
+
+```dvc
+$ dvc unprotect data.csv
+```
```

Comment:
Should this be a note (md block quote)? Since those types of links are never the default.

Reply:
Added this to a block quote. Can there be code blocks inside a block quote?

Reply:
Yes there can. I see this is done, thanks. Resolving